### ISSUE TITLE
All instances of "further" should be "farther"

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab-basics#83]
Apologies that I miss-spelled "farther" in several places in the design document.

Please change all instances of "further" to "farther".

This is of course, the same for both GFL:Basics and GFL regular. I think I have already noted the changes in an issue in the GFL repo.

**Comment by zepumph** [comment]
I saw three instances in the suite, changed above. Over to you @terracoda.

**Comment by terracoda** [comment]
@zepumph, I think you got all the "furthers" in both REGUKLAR and BASICS! Thanks for correcting this for me! 
Closing.
